src/agentlab/experiments/study.py

```diff
@@ -113,17 +178,14 @@ def run(self, n_jobs=1, parallel_backend="joblib", strict_reproducibility=False,
             raise ValueError("exp_args_list is None. Please set exp_args_list before running.")
 
         logger.info("Preparing backends...")
-        self.benchmark.prepare_backends()
+        try:
+            self.benchmark.prepare_backends()
+        except ReadTimeout:
+            logger.warning("Backend preparation timed out. Continuing anyway.")
+
```

Comment:
You can remove this now, webarena reset should not timeout any more
```suggestion
        self.benchmark.prepare_backends()

```